designs/2021-init-command-eslint-cli/README.md

```diff
@@ -0,0 +1,153 @@
+
+It can be removed by using the local installed eslint, and switch to the new `Linter` api.
+
+espree was used:
+https://github.com/aladdin-add/eslint-create/blob/c27509cbbaea7c846fd9d7a373feece5254ac8d8/lib/config-initializer.js#L161
+
+It can be removed after https://github.com/eslint/espree/issues/495 get landed.
+
+### 4. Release the new package
+
+### 5. Update eslint repo
+
+- Remove `eslint --init` related files.
+- the following dependencies can be removed in eslint repo:
+  - enquirer
+  - semver
+- Whenever `--init` command is being used, show a warning(e.g. "You can also run this command directly using 'npm init @eslint/config'") and run `npm init @eslint/config` using `child_process` of the native node modules.
+
+## Documentation
+
+<!--
+    How will this RFC be documented? Does it need a formal announcement
+    on the ESLint blog to explain the motivation?
+-->
+- [get-started](https://eslint.org/docs/user-guide/getting-started).
+- Documentation for `@eslint/create-config` will be created with proper usage and each prompt's details. In the main documentation, a link to `@eslint/create-config` will be given [here](https://github.com/eslint/eslint/blob/master/docs/user-guide/command-line-interface.md#--init) and basic usage and the package details will be documented.
+  Also in the cli command options, [here](https://github.com/eslint/eslint/blob/master/docs/user-guide/command-line-interface.md#options), for `--init` we need to change the description.
+- A formal announcement in not needed, as it is not a breaking change.
+
+## Drawbacks
+
+<!--
+    Why should we *not* do this? Consider why adding this into ESLint
+    might not benefit the project or the community. Attempt to think
+    about any opposing viewpoints that reviewers might bring up.
+
+    Any change has potential downsides, including increased maintenance
+    burden, incompatibility with other tools, breaking existing user
+    experience, etc. Try to identify as many potential problems with
+    implementing this RFC as possible.
+-->
+
+- This might increase the maintenance burden as this will add one more repo in the organization.
+- We may need to face challenges like re-directing of issues from `@eslint/create-config` repo to `eslint` and vice-versa.
+
+## Backwards Compatibility Analysis
+
+<!--
+    How does this change affect existing ESLint users? Will any behavior
+    change for them? If so, how are you going to minimize the disruption
+    to existing users?
+-->
+
+- Users can use `eslint --init`, which is as the same as current.
+- Users can no longer use auto-config.
+
+## Alternatives
+
+<!--
+    What other designs did you consider? Why did you decide against those?
+
+    This section should also include prior art, such as whether similar
+    projects have already implemented a similar feature.
+-->
+
+The current implementation in `eslint` repo is the alternative. No changes needed.
+
+## Open Questions
+
```

Comment:
Should we also update usage from `eslint --init` to `eslint init`?

Reply:
I didn't see why it should be changed. it seems out scope of the RFC.

Reply:
Yeah, it just crossed my mind as some other tools use `init` -> `git init`, `webpack init`, `npm init` etc.

Reply:
ah, I see! the new usage is `npm init @eslint/config`. so, it's no longer an issue. :)

Reply:
👍🏻